### ISSUE TITLE
Ambiguous region annotation

### ISSUE THREAD

**Issue by j-min** [jamespark3922/visual-comet#3]
Hi @jamespark3922 ,

Since the box region annotations are written in str instead of list, it's hard to parse which region each text description refers to.
For example, in the first row, `81 Event: 310 and 13 are acting on the stage in costumes 	# objects: 16`, how can we parse '310'? It can be both 3 + 10 and 3 + 1 + 0.

![image](https://user-images.githubusercontent.com/18069263/102802745-1cedf400-4385-11eb-96dd-040fb90229de.png)

**Comment by j-min** [comment]
I found visualcomet region annotations start from 1, since 0 corresponds to entire image. It would be nice to note this in readme since VCR region annotations start from 0.
By the way, could you confirm that all numbers in visualcomet text (event, place, inference text) correspond to VCR's 'person' region annotations (=metadata['names'] correspond to 'person') and the indices of region annotations are just shifted by 1?

**Comment by jamespark3922** [comment]
> Hi @jamespark3922 ,
> 
> Since the box region annotations are written in str instead of list, it's hard to parse which region each text description refers to.
> For example, in the first row, `81 Event: 310 and 13 are acting on the stage in costumes # objects: 16`, how can we parse '310'? It can be both 3 + 10 and 3 + 1 + 0.

Thank you for pointing this out! I've fixed the error when some annotators were using dash to denote various people. You can use the same link to download the annotations from the same link [https://storage.googleapis.com/ai2-mosaic/public/visualcomet/visualcomet.zip](https://storage.googleapis.com/ai2-mosaic/public/visualcomet/visualcomet.zip). It should be easy now to parse the strings as the person ids are all separated by space.

-----
> I found visualcomet region annotations start from 1, since 0 corresponds to entire image. 
> It would be nice to note this in readme since VCR region annotations start from 0.

VCR text starts annotations from 1 as well, which is why I use the same format in annotation text. When the text was tokenized, the ids are referenced starting from 0. This is based on your implementation choice and how you wish to tokenize the sentence, but I'll note this in readme.   

-----
> By the way, could you confirm that all numbers in visualcomet text (event, place, inference text) correspond to VCR's 'person' region annotations (=metadata['names'] correspond to 'person') and the indices of region annotations are just shifted by 1?

Correct, all the numbers in visualcomet text are 'person' annotations in vcr.
These refer to the same first person in vcr "objects" list
- VCR question_orig: "How is 1 feeling?"
- VCR question: ["How", "is", [0], "feeling", "?"]
- VisualComet text: "notice 1 walk in the door"

**Comment by j-min** [comment]
Thanks for clarifying and editing the annotations! Hope you had good holidays :)
